src/pages/workspace/categories/CategorySettingsPage.tsx

```diff
@@ -245,7 +245,12 @@ function CategorySettingsPage({
                                                 <Switch
                                                     isOn={policyCategory?.areCommentsRequired ?? false}
                                                     accessibilityLabel={translate('workspace.rules.categoryRules.requireDescription')}
-                                                    onToggle={() => Category.setPolicyCategoryDescriptionRequired(policyID, categoryName, !areCommentsRequired)}
+                                                    onToggle={() => {
+                                                        if (policyCategory.commentHint && areCommentsRequired) {
+                                                            Category.setWorkspaceCategoryDescriptionHint(policyID, categoryName, '');
+                                                        }
```

Comment:
@parasharrajat Did you detect any issues with `areCommentsRequired`? 

I used areCommentsRequired because setPolicyCategoryDescriptionRequired also uses areCommentsRequired. If you don’t agree with using areCommentsRequired, we can use isOn, which will still work well

https://github.com/Expensify/App/blob/faa48b605d708d7e8814e54fbf2b88a883c9c0c0/src/pages/workspace/categories/CategorySettingsPage.tsx#L248

https://github.com/Expensify/App/blob/05fd93d94bd78e5b9812a39972f2e9287c7437ae/src/libs/actions/Policy/Category.ts#L329

Reply:
@parasharrajat what do you think?

Reply:
@parasharrajat thank you!